scene/main/viewport.cpp

```diff
@@ -3745,10 +3721,8 @@ Viewport::Viewport() {
 	viewport_textures.insert(default_texture.ptr());
 	default_texture->proxy = RS::get_singleton()->texture_proxy_create(texture_rid);
 
-	//internal_listener_2d = SpatialSound2DServer::get_singleton()->listener_create();
```

Comment:
has been commented out 5 years ago in https://github.com/godotengine/godot/commit/b400c69cd487f70d8164dd5550eb994253d359d6